.github/workflows/lint.yml

```diff
@@ -21,6 +21,6 @@ jobs:
           git diff --exit-code go.mod
 
       - name: Lint
-        uses: golangci/golangci-lint-action@v3.1.0
+        uses: golangci/golangci-lint-action@v3.4.0
         with:
-          version: latest
+          version: v1.52
```

Comment:
Was there a reason not to pin this to the actual latest version?

Reply:
Is it cause they were released in the last 13 hours? 🤣

Reply:
No reason. It's just that I have verified that this version works.